### DISCUSSION TITLE
Pasting plain text produces error when using mention function

### DISCUSSION THREAD

**Discussion by redpandaex** [ueberdosis/tiptap#2413] [General Questions] [answered]
Using the mention function, when the plain text contains '@', the paste will generate an error, the specific error is as follows
![image](https://user-images.githubusercontent.com/95856798/150450306-b0aa3ea3-3965-482f-b2d2-eada410b5675.png)

**Comment by philippkuehn** [comment, marked as answer]
oh you are right. [this patch](https://github.com/ueberdosis/tiptap/commit/55fa2208996f648a2ca06b0a363a249ff5e39169) should help.

**Reply by redpandaex** [reply]
This may be very useful for my project, thank you
